src/components/Timelog/TimeEntryForm/TimeEntryForm.jsx

```diff
@@ -75,13 +75,14 @@ const TimeEntryForm = (props) => {
   const [isInfoModalVisible, setInfoModalVisible] = useState(false);
   const [projects, setProjects] = useState([]);
   const [tasks, setTasks] = useState([]);
+  const [formDataBeforeEdit, setFormDataBeforeEdit] = useState({});
 
   const fromTimer = !_.isEmpty(timer);
   const { userProfile, currentUserRole } = useSelector(getTimeEntryFormData);
 
   const dispatch = useDispatch();
 
-  const tangibleInfoToggle = (e) => {
+  const tangibleInfoToggle = e => {
```

Comment:
Did you delete the parenthesis in line 85 and 47 to be consistent in how you write the code?

Reply:
I think this is caused by the prettier auto re-format the file when I changed the code on this file.